tutorials/send-connect-notification-to-slack-from-google-compute-engine.md

```diff
@@ -66,18 +72,42 @@ If git is not installed, download and extract the code.
 
 ## Creating a Slack incoming webhook
 
-Create a [Slack incoming webhook](https://api.slack.com/incoming-webhooks) from
-the [custom integrations page for your Slack
-team](https://slack.com/apps/manage/custom-integrations). You should see a
-webhook URL, like
-`https://hooks.slack.com/services/T000000/B00000/XXXXXXXX`.
-
-Write this webhook URL to a file called `slack-hook` in the `notify` directory.
-
-    echo 'https://hooks.slack.com/services/T000000/B00000/XXXXXXXX' > slack-hook
+An [incoming webhook](https://api.slack.com/incoming-webhooks) creates an HTTPS
+endpoint where you can send messages. These messages will post the the
+configured channel or direct message.
+
+1.  Create a [new Slack app](https://api.slack.com/apps).
+    1.  Give the app a name, such as "SSH Notifier".
+    1.  Choose the Slack team where you want it installed.
+1.  Select the [Slack incoming
+    webhook](https://api.slack.com/incoming-webhooks) feature in the **Add
+    features and functionality** section.
+    1.  Click the **Off** switch in the upper right-hand corner to activate the
+        incoming webhooks feature.
```

Comment:
Maybe add that the switch should change to "on" or something? Seems odd clicking an "off" button to turn something _on_.